### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by HotzingTone** [JuliaConstraints/JuliaConstraints.github.io#52] [General]
Hi there, I am very interested in using Julia Constraints, but find from your website that the documentation is still incomplete. I wonder how long it would take to finish the documentation? Thank you very much indeed!

**Comment by Azzaare** [comment]
Hi!

We are making a big pass on a lot of core package this summer, and the documentation will progress alongside those changes.

I am not sure if I can ensure the deadline, but I hope to see thing wrapped up around the end of September. However, most of the bases should be available in the doc around the end of August.

**Reply by HotzingTone** [reply]
Thank you so much Azzaare for your hard work.
Really looking forward to it!
